jax/_src/numpy/lax_numpy.py

```diff
@@ -100,9 +100,6 @@ def canonicalize_shape(shape: Any, context: str="") -> core.Shape:
 e = np.e
 euler_gamma = np.euler_gamma
 inf = np.inf
-NINF = np.NINF
-PZERO = np.PZERO
```

Comment:
Out of curiosity, what should users replace `PZERO` and `NZERO` with? `0.0` and `-0.0`?

Reply:
Yes, in numpy's Python API, these should be used.

Reply:
It'd be helpful if there were a table somewhere of replacements (e.g., in the NumPy release notes?)

Reply:
Currently, it's in an issus, as this list still evolves: https://github.com/numpy/numpy/issues/24306#issuecomment-1663564574 (when you open "details" in this comment you will see current state of main namespace. I will decorate each "removed" with recommended replacement).

Yes, release notes will contain all changes that are being introduced right now.